src/Microsoft.CodeAnalysis.Razor.Workspaces/WorkspaceRegistrationMonitor.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.CodeAnalysis.Razor
+{
+    public class WorkspaceRegistrationMonitor
```

Comment:
This doesn't abstract anything. It's like, literally the same semantic as `WorkspaceRegistration`.